### ISSUE TITLE
Doesn't work correctly on Twitter

### ISSUE THREAD

**Issue by Warix3** [kimsey0/FirefoxAutoDict#3]
When the page loads (and if you write in one language and delete what you've written) the placeholder text in the input box for a new tweet is not checked. And it doesn't update the language while writing a tweet. 

For page load or deleting the content and focusing out of the box so the placeholder appears:
![image](https://user-images.githubusercontent.com/9502571/58839974-3543d400-8664-11e9-86bb-bca833db3fa5.png)

And the language doesn't get updated while typing:
![image](https://user-images.githubusercontent.com/9502571/58840036-7dfb8d00-8664-11e9-8a9d-6a72702dc750.png)

**Comment by rogerdahl** [comment]
Is it working for you on other sites? If so, which ones? What's your OS and browser versions?

**Comment by kimsey0** [comment]
It seems to work for me, but only changes the language when I remove focus from the text area by clicking somewhere else on the page, then clicking in the textarea again. @Warix3, could you check if the same is the case for you? Then I'll see if I can figure out why it doesn't change as you type.

**Comment by Warix3** [comment]
@kimsey0 Yes it works if I change the focus while typing. It doesn't automatically change the language as I type like on other sites. 
@rogerdahl it works on other sites, OS is windows 10 and browser firefox (latest version 67.0)

It's not just one issue, it's a couple of issues.
The first issue is with the placeholder text. If my twitter is in Croatian like in the first image then the placeholder text will be in Croatian. If I type in something in English and change focus to change the language and then delete the contents of the input box then the Croatian placeholder will appear but the spell checking will stay in English and it won't update for the placeholder. But I think this is twitter's fault because they are not using attribute placeholder but instead they are using actual value placeholder. But still maybe something can be done.
The second issue is the one for which you asked me about and it's the one where it doesn't change language automatically while you type but instead you need to focus in and out to change the language.
